packages/tools/src/eventListeners/mouse/index.ts

```diff
@@ -12,6 +14,14 @@ import mouseMoveListener from './mouseMoveListener';
  */
 function disable(element: HTMLDivElement): void {
   element.removeEventListener('dblclick', mouseDoubleClickListener);
+
+  // A separate double click listener for the element. Separate because...
+  // - it listens on the capture phase (and not the typical bubble phase)
+  // - the data used to ignore the double click is private to mouseDoubleClickIgnoreListener
+  element.removeEventListener('dblclick', mouseDoubleClickIgnoreListener, {
```

Comment:
This adds the listener for every enabled viewport. It is a bit of a waste since only one (global) listener is required.